docs/best_practices/tool_xml.rst

```diff
@@ -208,6 +208,17 @@ warning messages appear). See `planemo docs <https://planemo.readthedocs.io/en/l
 When tools contain output filters, tests should be included that verify
 this filtering occurs. See `planemo docs <https://planemo.readthedocs.io/en/latest/writing_how_do_i.html#test-output-filters-work>`__ for more information.
 
+Test data guidelines:
+
+* Test data should be as small as possible while still being representative.
+* Consider using subsets or downsampled versions of real datasets.
+* If larger test files are unavoidable, they should be deposited in
+  the `Galaxy Tool Developers <https://zenodo.org/communities/galaxy_tool_dev_sig>`__
+  Zenodo community rather than uploaded directly to
+  GitHub or stored in private Zenodo repositories.
+
+To upload test data to the Zenodo community:
```

Comment:
It would be nice if you can also show how a Zenodo dataset can be used in the Galaxy tool XML test.